doc-content/enterprise-only/openshift/hacep-server-proc.adoc

```diff
@@ -30,7 +30,7 @@ oc create -f kubernetes/role.yaml
 oc create -f kubernetes/role-binding.yaml
 ----
 +
-. In the `springboot` directory, enter the following commands to build the docker image and push into docker hub. This build imports the built `sample-hacep-project-kjar` code as a Maven dependency and includes it in the `BOOT-INF/lib` directory of the `openshift-kie-springboot.jar` fat jar file. The Docker build then uses the fat jar file to create the image.
+. In the `springboot` directory, enter the following commands to build the Docker image and push into Docker Hub. This build imports the built `sample-hacep-project-kjar` code as a Maven dependency and includes it in the `BOOT-INF/lib` directory of the `openshift-kie-springboot.jar` jar file. The Docker build then uses the jar file to create the image.
```

Comment:
Should be "JAR file" per our IBM/RH style standard (all caps for file abbreviations in-line). And actually, suggest removing the first one since it always has the extension in the file name, so "`....jar` file. The Docker ... the JAR file to ..."